packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -411,18 +414,27 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
         allowMinify &&
         (buildOptions.platform == 'server'
           ? {
-            ecma: terserEcma,
-            global_defs: angularGlobalDefinitions,
-            keep_fnames: true,
-          }
+              ecma: terserEcma,
+              global_defs: angularGlobalDefinitions,
+              keep_fnames: true,
+            }
           : {
-            ecma: terserEcma,
-            pure_getters: buildOptions.buildOptimizer,
-            // PURE comments work best with 3 passes.
-            // See https://github.com/webpack/webpack/issues/2899#issuecomment-317425926.
-            passes: buildOptions.buildOptimizer ? 3 : 1,
-            global_defs: angularGlobalDefinitions,
-          }),
+              ecma: terserEcma,
+              pure_getters: buildOptions.buildOptimizer,
+              // PURE comments work best with 3 passes.
+              // See https://github.com/webpack/webpack/issues/2899#issuecomment-317425926.
+              passes: buildOptions.buildOptimizer ? 3 : 1,
+              global_defs: angularGlobalDefinitions,
```

Comment:
Thinking about it, shouldn't `global_defs` and `pure_funcs` be configured when platform is server as well?